e2e/automated.test.js

```diff
@@ -69,4 +69,27 @@ describe('End to End tests', () => {
       });
     });
   });
+
+  describe('expanded blob', () => {
+    it('should resolve after appending new blobs', async () => {
+      const url =
+        'https://github.com/OctoLinker/OctoLinker/pull/451/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2';
+      const targetUrl = 'https://github.com/benmosher/eslint-plugin-import';
+      const selector = '[data-line-number="45"] + td .octolinker-link';
+      const expandSelector = '[data-right-range="44-55"]';
+
+      await page.goto(url);
+
+      await page.click(expandSelector);
+
+      await page.waitForSelector(selector);
+
+      await Promise.all([
```

Comment:
I think it's enough to check if OctoLinker insert new links into the dom without actually clicking them. Therefore I would suggest to remove everything between line 87-92.

Reply:
Right, i'll update it.